Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -346,21 +407,36 @@ void EnhancementsWidget::LoadPPShaders(StereoMode stereo_mode)
 
 void EnhancementsWidget::LoadSettings()
 {
-  m_block_save = true;
-  m_texture_filtering_combo->setEnabled(Config::Get(Config::GFX_HACK_FAST_TEXTURE_SAMPLING));
-  m_arbitrary_mipmap_detection->setEnabled(!Config::Get(Config::GFX_ENABLE_GPU_TEXTURE_DECODING));
+  m_texture_filtering_combo->setEnabled(ReadSetting(Config::GFX_HACK_FAST_TEXTURE_SAMPLING));
+  m_arbitrary_mipmap_detection->setEnabled(!ReadSetting(Config::GFX_ENABLE_GPU_TEXTURE_DECODING));
 
   // Anti-Aliasing
 
-  const u32 aa_selection = Config::Get(Config::GFX_MSAA);
-  const bool ssaa = Config::Get(Config::GFX_SSAA);
-  const int aniso = Config::Get(Config::GFX_ENHANCE_MAX_ANISOTROPY);
+  const u32 aa_selection = ReadSetting(Config::GFX_MSAA);
+  const bool ssaa = ReadSetting(Config::GFX_SSAA);
+  const int aniso = ReadSetting(Config::GFX_ENHANCE_MAX_ANISOTROPY);
   const TextureFilteringMode tex_filter_mode =
-      Config::Get(Config::GFX_ENHANCE_FORCE_TEXTURE_FILTERING);
+      ReadSetting(Config::GFX_ENHANCE_FORCE_TEXTURE_FILTERING);
 
+  const QSignalBlocker blocker_aa(m_aa_combo);
   m_aa_combo->clear();
+  std::vector<u32> aa_modes;
+
+  // Bypass global backend_info when making game-specific changes.
```

Comment:
Why is this needed?

Reply:
Yeah... I think it's not terrible. A way to poll the information would be better.